memorious/operations/aleph.py

```diff
@@ -59,10 +60,12 @@ def aleph_emit_document(context, data):
     source_url = data.get("source_url", data.get("url"))
     foreign_id = data.get("foreign_id", data.get("request_id", source_url))
     # Fetch document id from cache
-    document_id = context.get_tag(make_key(collection_id, foreign_id, content_hash))
-    if document_id:
+    document = context.get_tag(make_key(collection_id, foreign_id, content_hash))
+    if document:
```

Comment:
@simonwoerpel What do you think about adding a `isinstance(document, dict)` check here as a middle ground between a migration hack and forcing users to manually drop data in the tags table?
The old data will be silently ignored and overwritten since it won't pass the `isinstance` check. What do you think?

Reply:
yes sure that would be reasonable :)

Reply:
voila https://github.com/alephdata/memorious/pull/190/commits/5a6a23709db778bbcda0c7f95d450d87adaca977